src/states/formState.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { useState } from 'react';
+
+export const UseFormState = () => {
+  const [formState, setFormState] = useState({
+    title: '',
+    userValue: 0,
+  });
+
+  return [formState, setFormState] as const;
+};
```

Comment:
I would say this hook is redundant since it just returns useState values and doesn't encapsulates any additional logic.